frontend/ui/data/creoptions.lua

```diff
@@ -221,9 +221,8 @@ In the top menu → Settings → Status bar, you can choose whether the bottom m
                     G_defaults:readSetting("DCREREADER_CONFIG_T_MARGIN_SIZES_XX_HUGE"),
                 },
                 hide_on_apply = true,
-                name_text_hold_callback = function(configurable, opt, prefix)
-                    optionsutil.showValues(configurable, opt, prefix, nil, "mm")
-                end,
+                name_text_hold_callback = optionsutil.showValues,
+                name_text_unit = true,
```

Comment:
I don't think I've read a comment anywhere saying what `true` means.

Reply:
Will do. (It basically piggybacks on the `unit and unit ~= "pt" -> use user setting` check in optionsutil to mean "it's a dimension that isn't in pt, follow user settings")